eos/statistics/log-likelihood.cc

```diff
@@ -1330,7 +1330,7 @@ namespace eos
                 {
                     double llh = (*b)->evaluate();
                     if (! std::isfinite(llh))
-                        return llh;
+                        return -std::numeric_limits<double>::infinity();
```

Comment:
Is the minus sign in front of std::... ok?

Reply:
Yes.